xsd/NeTEx_publication.xsd

```diff
@@ -16,28 +16,39 @@
 				<Contributor>CEN TC278 WG3 SG9 Team.</Contributor>
```

Comment:
These changes are in my view already in one of the other PR. Here I think we can omit them. @nick-knowles or don't you agree?